pyanaconda/ui/tui/spokes/time_spoke.py

```diff
@@ -428,21 +428,23 @@ def apply(self):
         pass
 
 
-class AddNTPServerSpoke(EditTUIDialog):
+class AddNTPServerSpoke(NormalTUISpoke):
```

Comment:
I don't think it is a good idea. 
1) There is `RemoveNTPSpoke` which is almost the same so it would be inconsistent.
2) It behaves slightly different - you don't want to show existing values.

Reply:
Ok.